### ISSUE TITLE
I can not use Estimator from tf.estimator package  TF 1.3

### ISSUE THREAD

**Issue by lukashes** [tensorflow/transform#36]
I have this bundle:
* TF: 1.3.0
* TFT: 0.1.10
* Python: 2.7.12

I read TF 1.3 docs [here](https://www.tensorflow.org/programmers_guide/estimators) about Estimators:

> Note: TensorFlow also provides an Estimator class at tf.contrib.learn.Estimator, which you should not use.

I tried to change `learn.Estimator` with `tf.estimator.Estomator` and got error (all examples TFT works with `learn.Estimator`):

> local/lib/python2.7/site-packages/tensorflow/python/estimator/estimator.py", line 440, in export_savedmodel
>     serving_input_receiver.receiver_tensors,
> AttributeError: 'InputFnOps' object has no attribute 'receiver_tensors'

I use export_strategy for serving model through `tf.contrib.learn.Experiment` and it should support `tf.estimator.Estomator`, from docs:
> estimator: Object implementing Estimator interface, which could be a combination of ${tf.contrib.learn.Trainable} and ${tf.contrib.learn.Evaluable} (deprecated), or ${tf.estimator.Estimator}.

Can you explain what version of TFT/TF I should use, or what I am doing wrong?

**Comment by elmer-garduno** [comment]
To make sure things are fully compatible pleas use TFT version 0.3.0 which we just released yesterday.

https://github.com/tensorflow/transform/releases/tag/v0.3.0

That should help with the InputFnOps problem, please let me know if it works.

**Comment by lukashes** [comment]
Tried this bundle:
> tensorflow==1.3
> tensorflow_transform==0.3
> six==1.10.0

Got the same error:
> AttributeError: 'InputFnOps' object has no attribute 'receiver_tensors'

experiment fn:

 ```
        estimator = tf.estimator.LinearClassifier(feature_columns=[weighted_reviews],
                                         n_classes=LABEL_SIZE,
                                         model_dir=output_dir,
                                         config=tf.contrib.learn.RunConfig(save_checkpoints_steps=100))

        transformed_metadata = metadata_io.read_metadata(transformed_metadata_dir)
        raw_metadata = metadata_io.read_metadata(raw_metadata_dir)

        train_input_fn = input_fn_maker.build_training_input_fn(
          transformed_metadata,
          transformed_train_filepattern,
          training_batch_size=TRAIN_BATCH_SIZE,
          label_keys=[LABEL_COLUMN])

        eval_input_fn = input_fn_maker.build_training_input_fn(
          transformed_metadata,
          transformed_test_filepattern,
          training_batch_size=1,
          label_keys=[LABEL_COLUMN])

        serving_input_fn = input_fn_maker.build_default_transforming_serving_input_fn(
          raw_metadata=raw_metadata,
          transform_savedmodel_dir=transform_fn_dir + "/transform_fn",
          raw_label_keys=[],
          raw_feature_keys=[KEYWORDS_COLUMN])

        export_strategy = saved_model_export_utils.make_export_strategy(
          serving_input_fn,
          exports_to_keep=5,
          default_output_alternative_key=None)

        max_steps = MAX_STEPS if MAX_STEPS > 0 else \
          TRAIN_NUM_EPOCHS * NUM_TRAIN_INSTANCES / TRAIN_BATCH_SIZE

        return tf.contrib.learn.Experiment(
          estimator=estimator,
          train_steps=max_steps,
          eval_steps=NUM_TEST_INSTANCES,
          train_input_fn=train_input_fn,
          eval_input_fn=eval_input_fn,
          export_strategies=export_strategy,
          min_eval_frequency=5)
```

**Comment by elmer-garduno** [comment]
I see, yes we need to upload a fix to support the core estimators, will update this bug with more info as soon as we push it to github.

**Comment by davidsoergel** [comment]
Yep, that's a known issue; will fix soon.

**Comment by KesterTong** [comment]
Fixed in 0.3.1
